src/Umbraco.Web.UI.Client/src/less/components/application/umb-app-header.less

```diff
@@ -2,12 +2,65 @@
     background: @blueExtraDark;
     display: flex;
     align-items: center;
-    justify-content: space-between;
     max-width: 100%;
     height: @appHeaderHeight;
     padding: 0 20px;
 }
 
+.umb-app-header__logo {
+    margin-right: 30px;
+    button {
+        img {
+            height: 30px;
+        }
+    }
+}
+
+.umb-app-header__logo-modal {
+    position: absolute;
+    z-index: 7500;
```

Comment:
@nielslyngsoe I think we should place this together with most other variables regarding z-indexes.
See also this regarding z-index of notifications and overlays.

Reply:
@bjarnef thanks, good catch